### PR TITLE
Drone io sh

### PR DESCRIPTION
**Pull request by adam-singer**
Unable to test cause dartdoc-viewer is broken for dev channel sdk. Lets merge this PR and wire up the drone.io part when dartdoc-viewer is fixed. Bascially you dont want too much configuration in the drone io site, its easy to have a single shell script that is called from drone.io which drives all other tests to run.

### CODE REVIEW

File: drone_io.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+rm dartdoc-viewer/client/out/web/static/css/packages
+
+mv dartdoc-viewer/client/out/packages dartdoc-viewer/client/out/web/packages
+mv dartdoc-viewer/client/out/web ./.docs_staging
+
+# TODO: run pub build web.. This will transform Pixelate and proper packages folder
+pub build web 
+
+# TODO: stage web
+cp -r web .web_staging
```
Comment (donny-dont) [inline drone_io.sh]: Is this correct? Pub will build web to build/web. Also since its generated you could probably get away with a mv.

Comment (adam-singer) [inline drone_io.sh]: yep, we could use mv, wouldn't really matter. Once dartdoc-viewer is working again then we can adjust this script as needed.

Comment (adam-singer) [inline drone_io.sh]: Actually should bring in better shell scripting to find the packages links... memories of the older "deploy" dart scripts..

Comment (donny-dont) [inline drone_io.sh]: So after you build you should have a build/web and that'll have build/web/packages/ and all the JS that's needed.

Comment (adam-singer) [inline drone_io.sh]: k, that makes sense, will update

**Pull request merged**